java-generator/it/src/it/plugin/gradle/simple/build.gradle

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java library project to get you started.
+ * For more details take a look at the 'Building Java & JVM projects' chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/8.1/userguide/building_java_projects.html
+ */
+
+plugins {
+    // Apply the java-library plugin for API and implementation separation.
+    id 'java-library'
+    id 'io.fabric8.java-generator' version "${kubernetesClientVersion}"
+}
+
+repositories {
+    // Use Maven Central for resolving dependencies.
+    mavenCentral()
+    mavenLocal()
+}
+
+sourceSets {
+  main {
+    java {
+      srcDirs "$buildDir/generated/sources"
+    }
+  }
+}
+
+dependencies {
+  implementation  "io.fabric8:kubernetes-client:${kubernetesClientVersion}"
+  implementation  "io.fabric8:generator-annotations:${kubernetesClientVersion}"
+}
+
+// Apply a specific Java toolchain to ease working on different environments.
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(11)
+    }
+}
+
+javaGen {
+    urls = [
+      "https://raw.githubusercontent.com/artemiscloud/activemq-artemis-operator/v1.0.11/deploy/crds/broker_activemqartemis_crd.yaml",
```

Comment:
So far, we have attempted to keep the build and test "local", would it be possible to commit one CRD and test against it?

Reply:
Yep, done!